src/util/defaults.ml

```diff
@@ -135,6 +136,7 @@ let _ = ()
       ; reg Analyses "ana.sv-comp.functions" "false" "Handle SV-COMP __VERIFIER* functions"
       ; reg Analyses "ana.specification"   "" "SV-COMP specification (path or string)"
       ; reg Analyses "ana.wp"              "false" "Weakest precondition feasibility analysis for SV-COMP violations"
+      ; reg Analyses "ana.oct_no_uints"    "false"  "Use OctApron without tracking unsigned integers."
```

Comment:
I thought the problem is that now they will much easier go to top because I forget about them if they might be out of the bounds for their type, but just forgetting about them by default is not any better. Am I wrong?

Reply:
If the value `false` is sound and doesn't cause any other issues, then it's fine. I thought I noticed some other place, where this option was explicitly set to `true`, but I might be wrong.